mappers/gige-sdk/rc_genicam_api/set_value.cpp

```diff
@@ -0,0 +1,31 @@
+#include "set_value.h"
+
+extern "C" int set_value(MyDevice myDevice, const char* key, const char* value, char** err) {
+	int ret = SUCCESS;
+	std::string e = "";
+	*err = (char*)malloc(e.length() + 1);

```

Comment:
Put this code to the "try" {}. Remove those "free" in the "catch". Check NULL if malloc failed.